user-operator/src/test/java/io/strimzi/operator/user/operator/KafkaUserOperatorTest.java

```diff
@@ -933,6 +932,89 @@ public void testReconcileNewScramShaUser(VertxTestContext context)    {
             })));
     }
 
+    @Test
+    public void testReconcileNewScramShaUserwithConfigurableLength(VertxTestContext context)    {
```

Comment:
This looks good. I wonder if it would make more sense to focus on the password only and remove the assertion of the other things such as ACLs - since we would need to maintain those parts even when the test really tests the password length. But LG otherwise.